### PR TITLE
chore: upgrade conventional commits

### PR DESCRIPTION
**Pull request by icco**
This closes the following:

 - PRs: #632, #633, #634, #635, #636, #637, #638, #639
 - As suggested by https://github.com/conventional-changelog/conventional-changelog/issues/1267

### CODE REVIEW

**Review by dededecline — APPROVED** [review]

**Comment by travi** [comment]
Please make sure our tests are passing. `npm test` should be enough to verify this locally.

File: package.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
-    "conventional-changelog-ember": "4.0.0",
-    "conventional-changelog-eslint": "5.0.0",
-    "conventional-changelog-express": "4.0.0",
-    "conventional-changelog-jshint": "4.0.0",
+    "conventional-changelog-atom": "^5.0.0",
+    "conventional-changelog-conventionalcommits": "^8.0.0",
+    "conventional-changelog-ember": "^5.0.0",
+    "conventional-changelog-eslint": "^6.0.0",
+    "conventional-changelog-express": "^5.0.0",
+    "conventional-changelog-jshint": "^5.0.0",
```
Comment (travi) [inline package.json]: Please keep dev dependencies pinned to specific versions

**Review by travi — CHANGES_REQUESTED** [review]

**Comment by travi** [comment]
> Please make sure our tests are passing. `npm test` should be enough to verify this locally.

@icco the tests continue to fail after your follow up to pin the dev dependencies. are you interested in working through the breaking changes related to these updates in order to help us move this forward?

**Comment by icco** [comment]
> > Please make sure our tests are passing. `npm test` should be enough to verify this locally.
> 
> @icco the tests continue to fail after your follow up to pin the dev dependencies. are you interested in working through the breaking changes related to these updates in order to help us move this forward?

Yeah been trying to get the tests pass locally with no luck. I believe there's an underlying bug in the import plugin you use but can not figure it out.

**Comment by travi** [comment]
appreciate the update and the investigation. we are starting to investigate as well, but unsure how much time we can dedicate soon. please feel free to keep us up to date if you make progress, even if that means pushing wip changes that dont get to a point of getting tests to a passing state.

it could also be helpful to start with running the integration tests (`npm run test:integration`) first for a larger scope feedback loop before getting distracted by the unit tests (`npm run test:unit`) if the interface changes are unclear.

**Comment by travi** [comment]
appreciate the investment here, but closing in favor of https://github.com/semantic-release/release-notes-generator/pull/643. we could still use help updating the commit-analyzer plugin in similar ways in case you are still interested and the other PR helps with any complexities that you encountered with this effort

**Pull request closed without merging**